frameworks/TypeScript/bun/src/index.ts

```diff
@@ -0,0 +1,16 @@
+type HandlerFn = (req?: Request) => Promise<Response> | Response;
+
+const options: ResponseInit = { headers: { "Server": "Bun" } };
+
+const handlers = new Map<string, HandlerFn>();
+handlers.set("/json", () => Response.json({ message: "Hello, World!" }, options));
+handlers.set("/plaintext", () => new Response("Hello, World!", options));
```

Comment:
out of curiosity, shouldn't this be `Response.text("Hello, World!", options)` instead?

Reply:
> out of curiosity, shouldn't this be Response.text("Hello, World!", options) instead?

Not really, `new Response()` returns `text/plain` as content type already, required by the `/plaintext` scenario.

> or better, what is Response.json in there?

`Response.json` returns the object serialized and `application/json` as content type, required by the `/json` scenario.

Reply:
not in the docs and not standard though, maybe a comment would do

Reply:
This is the documentation for `Response.json`: https://developer.mozilla.org/en-US/docs/Web/API/Response/json_static

And I believe `text/plain` is the default content type.

Do you think an extra comment is really needed?

Reply:
I’ve missed that … my bad. No comment needed